### PR TITLE
sdk(resources): update-mobile-rnsdk-version script

### PR DESCRIPTION
**Pull request by Calinteodor**

### CODE REVIEW

File: resources/update-mobile-rnsdk-version.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+sed -i "" -e "s/${CURRENT_VERSION}/${VERSION}/" ${THIS_DIR}/../react-native-sdk/package.json
+
+npm install --prefix ${THIS_DIR}/../react-native-sdk
+
+git add .
+
+git commit -m "update version to ${VERSION}"
+
+git push upstream --force
```
Comment (saghul) [inline resources/update-mobile-rnsdk-version.sh:22]: Don't --force, or specify the remote, just `git push`

File: resources/update-mobile-rnsdk-version.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+if [[ $# -ne 1 ]]; then
+    echo "Please specify a version"
+    exit 1
+fi
+
+THIS_DIR=$(cd -P "$(dirname "$(readlink "${BASH_SOURCE[0]}" || echo "${BASH_SOURCE[0]}")")" && pwd)
+CURRENT_VERSION=$(node -p "require('${THIS_DIR}/../react-native-sdk/package.json').version")
+VERSION=$1
+
+sed -i "" -e "s/${CURRENT_VERSION}/${VERSION}/" ${THIS_DIR}/../react-native-sdk/package.json
```
[the remark this answers is not in the document]
Comment (Calinteodor) [reply resources/update-mobile-rnsdk-version.sh:14]: I tried this but it updates the central package.json and not the react-native-sdk one.

[the remark this answers is not in the document]
Comment (Calinteodor) [reply resources/update-mobile-rnsdk-version.sh:14]: Nevermind. I fixed it.

File: resources/update-mobile-rnsdk-version.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+set -e -u
+
+if [[ $# -ne 1 ]]; then
+    echo "Please specify a version"
+    exit 1
+fi
+
+THIS_DIR=$(cd -P "$(dirname "$(readlink "${BASH_SOURCE[0]}" || echo "${BASH_SOURCE[0]}")")" && pwd)
+CURRENT_VERSION=$(node -p "require('${THIS_DIR}/../react-native-sdk/package.json').version")
```
Comment (saghul) [inline resources/update-mobile-rnsdk-version.sh:11]: unused

File: resources/update-mobile-rnsdk-version.sh [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+VERSION=$1
+
+npm version --prefix ${THIS_DIR}/../react-native-sdk "${VERSION}" --no-git-tag-version
+
+npm install --prefix ${THIS_DIR}/../react-native-sdk
+
+git add .
+
+git commit -m "update version to ${VERSION}"
```
Comment (saghul) [inline resources/update-mobile-rnsdk-version.sh:20]: update rnsdk version...

**Review by saghul — APPROVED** [review]

**Pull request merged**